global.json

```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
-    "dotnet": "2.2.104"
+    "dotnet": "3.1.100"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Build.Tasks.Feed": "2.2.0-beta.19151.1",
-	"Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19320.1"
+    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20052.1",
+    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20052.1"
```

Comment:
This is an interesting bump for a version.
@JohnTortugo Can you please help us understand why that is?

Reply:
Are we aligning with .Net 5 ?

Reply:
I asked about that: "The update to 5 was due to the release of .net 3.1"